.devcontainer/Dockerfile

```diff
@@ -80,3 +81,8 @@ ENV TIZEN_ROOTFS /tizen_rootfs
 
 # Fast Model GDB plugins path for debugging support
 ENV FAST_MODEL_PLUGINS_PATH /opt/FastModelsPortfolio_11.16/plugins/Linux64_GCC-9.3
+
+# icecc and ccache for cluster compilation
+ENV CHIP_PW_COMMAND_LAUNCHER ccache
+ENV CCACHE_PREFIX icecc
```

Comment:
What if `iceccd` is not started? Does project compile without any warnings/errors?

Reply:
When `icecc` is installed but not running it will fallback to local compilation.